### ISSUE TITLE
Label needed on "scroll to top" button on News Feed

### ISSUE THREAD

**Issue by gmeben** [MESH-Research/knowledge-commons-profiles#378]
Area of site: News Feed
URL: https://hcommons-staging.org/activity/

**Description**
At the bottom right of the page, there is a "scroll to top" button. It has an icon with an upward-pointing arrow. It does not have a label or discernible text.

**Recommendation**

Either one of these will fix the issue: 

1. Add a `title` attribute to the button
2. Add an `aria-label` attribute to the button
